src/remote-proofs.js

```diff
@@ -61,6 +68,54 @@ class RemoteProofs {
     }
   }
 
+  formatRedditUrl (url) {
+    if (!validUrl.isUri(url)) {
+      throw new Error('Valid url is required')
+    }
+
+    let parsed = parse(url)
+    if (!parsed.hostname === 'www.reddit.com') {
+      throw new Error('Url has to use www.reddit.com as host')
+    }
+
+    if (url.endsWith('.json')) {
+      return url
+    } else if (url.endsWith('/')) {
+      let urlArr = url.split('/')
+      urlArr.pop() // removes the enpty
+      return `${urlArr.join('/')}.json`
+    } else {
+      throw new Error('Url is not correct format for a Reddit proof')
+    }
+  }
+
+  async getRedditData (url) {
+    let rUrl = this.formatRedditUrl(url)
+
+    if (!validUrl.isUri(rUrl)) {
+      throw new Error('Valid url is required')
+    }
+    try {
+      return await fetch(rUrl)
```

Comment:
This `await` is unnecessary. Since you're returning a promise anyway there's no need to block on this, especially if this function is every called in a `Promise.all`. It might make sense to add [no-return-await](https://eslint.org/docs/rules/no-return-await) to the eslint config

Reply:
I sometimes leave them in there on accident while testing stuff. Thanks! I bet there ar emore errant `awaits` throughout. I do need to add an issue to set up standardjs linting, etc. Please keep reviewing! :)

Reply:
I always take a look when something pops up on my github notifications, this is a super cool project 👍

Reply:
I really respect your skills and look forward to your eyeballs on this